File: BackEnd/ExamManagement/DPD.HR.Business.Dto/Dto/WorkflowTemplateDto.cs
using DPD.HumanResources.Entities.Entities;

namespace DPD.HumanResources.Dtos.Dto;

public class WorkflowTemplateDto
{
    public Guid Id {get;set;}
    public string? WorkflowName {get;set;}
    public string WorkflowCode {get;set;}
    public int Order {get;set;}
    public string? StartWorkflowButton {get;set;}
    public string? DefaultCompletedStatus {get;set;}
    public string TableName {get;set;}
    public DateTime CreatedDate {get;set;}
    public int? Status {get;set;}
    
    // public List<WorkflowInstancesDto> WorkflowInstancesList { get; set; }
    // public List<WorkflowStepDto> WorkflowSteps { get; set; }
}

File: FrontEnd/src/models/workflowTemplateDTO.ts
export interface WorkflowTemplateDTO {
  id: string;
  workflowName: string | null;
  workflowCode: string;
  order: number;
  startWorkflowButton: string | null;
  defaultCompletedStatus: string | null;
  tableName: string;
  createdDate: Date;
  status: number | null;
}
